configs/rt_slam_plugins.yaml

```diff
@@ -5,11 +5,11 @@ plugin_group:
   - name: Kimera-VIO/
     path:
       git_repo: https://github.com/ILLIXR/Kimera-VIO.git
-      version: "3.1"
+      version: "3.2"
   # - name: OpenVINS
   #   path:
   #     git_repo: https://github.com/ILLIXR/open_vins.git
-  #     version: "3.0"
+  #     version: "issue-58-standardize-clock"
```

Comment:
Is this [branch](https://github.com/ILLIXR/open_vins/pull/10) still needed? We merged `issue-58` into a merged branch,

Reply:
Yes, because since then, we've added commits to `issue-58-...` which are not in `master`. See [OpenVINS #14](https://github.com/ILLIXR/open_vins/pull/14).

Reply:
Merge ILLIXR/open_vins#14, push a new minor version tag, update here.